crypto/x509/x509_lu.c

```diff
@@ -651,32 +653,36 @@ X509_OBJECT *X509_OBJECT_retrieve_match(STACK_OF(X509_OBJECT) *h,
 int X509_STORE_CTX_get1_issuer(X509 **issuer, X509_STORE_CTX *ctx, X509 *x)
```

Comment:
Having looked around, I'm convinced this one is misnamed and should be called `X509_STORE_get1_issuer`.  If noone minds waiting a little bit, I'll ask DrH.